SocialCareCaseViewerApi/V1/Gateways/DatabaseGateway.cs

```diff
@@ -880,6 +880,18 @@ public List<Person> GetPersonsByListOfIds(List<long> ids)
             return _databaseContext.Persons.Where(x => ids.Contains(x.Id)).ToList();
         }
 
+        public Person GetPersonWithPersonalRelationshipsByPersonId(long personId)
+        {
+            var personWithRelationships = _databaseContext
+                .Persons
+                .Include(person => person.PersonalRelationships)
+                .FirstOrDefault(p => p.Id == personId);
+
+            personWithRelationships.PersonalRelationships = personWithRelationships.PersonalRelationships.Where(pr => pr.EndDate == null).ToList();
```

Comment:
This will throw an exception if the person doesn't exist

Reply:
Related to your above comment, I've added a check before this to ensure person isn't `null` so this doesn't happen anymore.